<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>SetupUsersWidget</class>
  <widget class="QWidget" name="SetupUsersWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>667</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Form1</string>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="1" >
        <spacer name="spacer9" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item row="1" column="2" >
        <spacer name="spacer12" >
          <property name="sizeHint" >
            <size>
              <width>61</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="1" column="0" >
        <spacer name="spacer12_2" >
          <property name="sizeHint" >
            <size>
              <width>61</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="2" column="1" >
        <widget class="QLabel" name="pass_error" >
          <property name="text" >
            <string/>
          </property>
          <property name="textFormat" >
            <enum>Qt::RichText</enum>
          </property>
          <property name="alignment" >
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="6" column="1" >
        <spacer name="spacer11" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item row="5" column="2" >
        <spacer name="spacer14" >
          <property name="sizeHint" >
            <size>
              <width>40</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="3" column="1" >
        <widget class="QLabel" name="caps_error" >
          <property name="text" >
            <string/>
          </property>
          <property name="textFormat" >
            <enum>Qt::RichText</enum>
          </property>
          <property name="alignment" >
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="5" column="0" >
        <spacer name="spacer14_2" >
          <property name="sizeHint" >
            <size>
              <width>40</width>
              <height>20</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Qt::Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item row="1" column="1" >
        <widget class="Q3ButtonGroup" name="buttonGroup1" >
          <property name="title" >
            <string>New User</string>
          </property>
          <layout class="QGridLayout" >
            <item row="0" column="0" >
              <widget class="QLabel" name="textLabel1_3" >
                <property name="text" >
                  <string>User Name:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="0" >
              <widget class="QLabel" name="textLabel1" >
                <property name="text" >
                  <string>Password (again):</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="0" >
              <widget class="QLabel" name="textLabel1_2" >
                <property name="text" >
                  <string>Password:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="1" >
              <widget class="QLineEdit" name="pass1" >
                <property name="echoMode" >
                  <enum>QLineEdit::Password</enum>
                </property>
              </widget>
            </item>
            <item row="1" column="0" >
              <widget class="QLabel" name="textLabel1_4" >
                <property name="text" >
                  <string>Real Name:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="1" >
              <widget class="QLineEdit" name="pass2" >
                <property name="echoMode" >
                  <enum>QLineEdit::Password</enum>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QLineEdit" name="username" />
            </item>
            <item rowspan="1" row="4" column="0" colspan="3" >
              <widget class="QCheckBox" name="admin" >
                <property name="text" >
                  <string>Give administrator privileges to this user</string>
                </property>
              </widget>
            </item>
            <item row="1" column="1" >
              <widget class="QLineEdit" name="realname" />
            </item>
            <item row="3" column="2" >
              <widget class="QPushButton" name="createButton" >
                <property name="text" >
                  <string>Create User</string>
                </property>
              </widget>
            </item>
            <item rowspan="3" row="0" column="2" colspan="1" >
              <widget class="QLabel" name="pix" >
                <property name="text" >
                  <string/>
                </property>
                <property name="alignment" >
                  <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="4" column="1" >
        <widget class="QLabel" name="textLabel2" >
          <property name="text" >
            <string>Users:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item row="5" column="1" >
        <layout class="QGridLayout" >
          <item rowspan="2" row="0" column="0" colspan="1" >
            <widget class="Q3ListBox" name="userList" />
          </item>
          <item row="1" column="1" >
            <spacer name="spacer8" >
              <property name="sizeHint" >
                <size>
                  <width>21</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
          <item row="0" column="1" >
            <layout class="QVBoxLayout" >
              <item>
                <widget class="QPushButton" name="deleteButton" >
                  <property name="text" >
                    <string>Delete Selected User</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="autoLoginLabel" >
                  <property name="text" >
                    <string>Auto Login User :</string>
                  </property>
                  <property name="wordWrap" >
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QComboBox" name="autoLogin" >
                  <item>
                    <property name="text" >
                      <string>Auto Login Disabled</string>
                    </property>
                  </item>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>username</tabstop>
    <tabstop>realname</tabstop>
    <tabstop>pass1</tabstop>
    <tabstop>pass2</tabstop>
    <tabstop>admin</tabstop>
    <tabstop>createButton</tabstop>
    <tabstop>userList</tabstop>
    <tabstop>deleteButton</tabstop>
    <tabstop>autoLogin</tabstop>
  </tabstops>
</ui>
